packages/datagovmy-ui/src/data-catalogue/Show/metadata.tsx

```diff
@@ -137,26 +154,62 @@ const DCMetadata: FunctionComponent<MetadataProps> = ({
             {/* URLs to dataset */}
             <div className="space-y-3">
               <h5>{t("meta_url")}</h5>
-              <ul className="text-dim ml-6 list-outside list-disc">
-                {Object.entries({
-                  csv: metadata.link_csv,
-                  parquet: metadata.link_parquet,
-                }).map(([key, url]: [string, unknown]) =>
-                  url ? (
-                    <li key={url as string}>
-                      <a
-                        href={url as string}
-                        className="text-primary dark:text-primary-dark break-all [text-underline-position:from-font] hover:underline"
-                        onClick={() =>
-                          track(key === "link_geojson" ? "csv" : (key as "parquet" | "csv"))
-                        }
-                      >
-                        {url as string}
-                      </a>
-                    </li>
-                  ) : undefined
-                )}
-              </ul>
+              {metadata.link_editions && metadata.link_editions.length > 0 ? (
```

Comment:
Can create a const like `hasEdition` to replace this check since it's used a lot.
Following code repeats the links section, better to reduce repeated blocks of code to help make refactoring easier in future.
Can do this:
- for dropdown, show only if hasEdition is true,
- for the links, use `getUrl()` function if hasEdition is true

Reply:
noted, working on it